src/action.cpp

```diff
@@ -540,6 +540,33 @@ action_id handle_action_menu()
     // Weight >= 200: Special action only available right now
     std::map<action_id, int> action_weightings;
 
+    // Check if we're in a potential combat situation, if so, sort a few actions to the top.
+    {
+        std::vector<Creature *> enemies_visible = g->u.get_hostile_creatures();
```

Comment:
You made a new scope, probably to limit the visibility of `enemies_visible`, which is good. But you're only using it once, you can use it directly without storing the data in a variable:
```C++
    if( !g->u.get_hostile_creatures().empty() ) {
```
That gives you a whole level of indentation. Also, please use `std::vector::empty()`, you do not care about the size, so don't query the size.